website/docs/capabilities/aitp-01-payments/aitp-01-payments.md

```diff
@@ -0,0 +1,333 @@
+# AITP-01: Payments
+
+* **Version**: 1.0.0
+* **Spec Status**: Ideation
+* **Implementation Status**: In Development
+
+:::warning Ideation Phase
+AITP-01 has a `schema.json` file, but the implementation is still being refined. The schema may evolve based on feedback and practical implementation experience. See the [Schema Reference](./schema-reference.md) for details on the current schema.
+:::
+
+## Overview
+
+AITP-01 Payments provides a purpose-built payment protocol for user-to-agent and agent-to-agent transactions. Unlike traditional payment systems that rely on credit cards or direct blockchain transactions, AITP-01 is designed specifically for the agent ecosystem, supporting complex payment flows, metered services, and fair compensation for all participating agents.
```

Comment:
all participating *parties ?

Not necessarily only agents that facilitate some transaction

Reply:
Fixed in https://github.com/nearai/aitp/pull/31/commits/fde085f2d6870fea666b8c302681b809e953b2ba.